frontend/src/index.css

```
@@ -1 +1,270 @@
+}
+
+/* ── Semantic tier — shadcn slots mapped to the primitive tier. ──────────────
+ * Mode-invariant: the same mappings are declared in :root and .dark; dark
+ * mode swaps the --ctp-* primitives above, the mappings below do not change.
+ * They are re-declared in .dark only so a reader sees the full set in one
+ * place. --radius is shadcn's default; --font-* are system stacks. */
+:root {
+  --radius: 0.625rem;
+
+  --font-sans:
+    ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Stylelint (17.12.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)